src/github.com/getlantern/yamlconf/yamlconf_test.go

```diff
@@ -214,69 +213,6 @@ func TestCustomPoll(t *testing.T) {
 	}, updated, "Custom polled config should contain correct data")
 }
 
-func TestConfigServer(t *testing.T) {
-	file, err := ioutil.TempFile("", "yamlconf_test_")
-	if err != nil {
-		t.Fatalf("Unable to create temp file: %s", err)
-	}
-	defer func() {
-		if err := os.Remove(file.Name()); err != nil {
-			t.Fatalf("Unable to remove file: %s", err)
-		}
-	}()
-
-	m := &Manager{
-		EmptyConfig: func() Config {
-			return &TestCfg{}
-		},
-		FilePath:         file.Name(),
-		FilePollInterval: pollInterval,
-		ConfigServerAddr: ConfigSrvAddr,
-	}
-
-	_, err = m.Init()
-	if err != nil {
-		t.Fatalf("Unable to init manager: %s", err)
-	}
-	m.StartPolling()
-
-	newNested := &Nested{
-		S: "900",
-		I: 900,
-	}
-	nny, err := yaml.Marshal(newNested)
-	if err != nil {
-		t.Fatalf("Unable to marshal new nested into yaml: %s", err)
-	}
-
-	_, err = http.Post(fmt.Sprintf("http://%s/N", ConfigSrvAddr), "text/yaml", bytes.NewReader(nny))
-	assert.NoError(t, err, "POSTing to config server should succeed")
-
-	updated := m.Next()
-
-	assert.Equal(t, &TestCfg{
-		Version: 2,
-		N:       newNested,
-	}, updated, "Nested should have been updated by POST")
-
-	req, err := http.NewRequest("DELETE", fmt.Sprintf("http://%s/N/I", ConfigSrvAddr), bytes.NewReader(nny))
-	if err != nil {
-		t.Fatalf("Unable to construct DELETE request: %s", err)
-	}
-	_, err = (&http.Client{}).Do(req)
-	assert.NoError(t, err, "DELETEing to config server should succeed")
-
-	updated = m.Next()
-
-	assert.Equal(t, &TestCfg{
-		Version: 3,
-		N: &Nested{
-			S: newNested.S,
-			I: FIXED_I,
-		},
-	}, updated, "Nested I should have reverted to default value after clearing")
-}
-
```

Comment:
This was testing old functionality that's now stripped. This should have been removed in that ticket but somehow slipped through.